export_test.go

```diff
@@ -0,0 +1,44 @@
+package filclient
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/urfave/cli/v2"
+)
+
+func TestExportFile(t *testing.T) {
+	outputFilename := "TestExportFile.test.output"
+	// Remove any pre-existing file and ensure it is removed when the test finishes
+	os.Remove(outputFilename)
+	defer func() {
+		os.Remove(outputFilename)
+	}()
```

Comment:
@elijaharita This seems kind of awkward, is it okay? Do you know of some other builtin way to test functions that output to files, and automatically clean up the files in Go?

Reply:
you can use `t.TempDir()` to get a new empty test directory generated by go which is automatically cleaned up after test finishes running

Reply:
so `outputFilename` would become `path.Join(t.TempDir(), "TestExportFile.test.output")`

what's the purpose of the output file naming scheme you used? is there any reason to do that over something simpler like just `export-test`?